cypress/fixtures/search-results-page-2.json

```diff
@@ -0,0 +1 @@
gg","preview-hq-mp3":"https://freesound.org/data/previews/399/399640_7386498-hq.mp3"},"num_downloads":91,"num_ratings":1},{"id":458002,"name":"Me Saying \"Hello\"","duration":2.50942,"username":"Fabrizio84","previews":{"preview-lq-ogg":"https://freesound.org/data/previews/458/458002_2841496-lq.ogg","preview-lq-mp3":"https://freesound.org/data/previews/458/458002_2841496-lq.mp3","preview-hq-ogg":"https://freesound.org/data/previews/458/458002_2841496-hq.ogg","preview-hq-mp3":"https://freesound.org/data/previews/458/458002_2841496-hq.mp3"},"num_downloads":365,"num_ratings":3},{"id":435851,"name":"Hello Scream Male Loud","duration":0.731429,"username":"dersuperanton","previews":{"preview-lq-ogg":"https://freesound.org/data/previews/435/435851_1992856-lq.ogg","preview-lq-mp3":"https://freesound.org/data/previews/435/435851_1992856-lq.mp3","preview-hq-ogg":"https://freesound.org/data/previews/435/435851_1992856-hq.ogg","preview-hq-mp3":"https://freesound.org/data/previews/435/435851_1992856-hq.mp3"},"num_downloads":1126,"num_ratings":7},{"id":399636,"name":"Hello #1.MP3","duration":2.548,"username":"SoundsForHim","previews":{"preview-lq-ogg":"https://freesound.org/data/previews/399/399636_7386498-lq.ogg","preview-lq-mp3":"https://freesound.org/data/previews/399/399636_7386498-lq.mp3","preview-hq-ogg":"https://freesound.org/data/previews/399/399636_7386498-hq.ogg","preview-hq-mp3":"https://freesound.org/data/previews/399/399636_7386498-hq.mp3"},"num_downloads":709,"num_ratings":3},{"id":123346,"name":"hello.wav","duration":1.674,"username":"MatteusNova","previews":{"preview-lq-ogg":"https://freesound.org/data/previews/123/123346_495253-lq.ogg","preview-lq-mp3":"https://freesound.org/data/previews/123/123346_495253-lq.mp3","preview-hq-ogg":"https://freesound.org/data/previews/123/123346_495253-hq.ogg","preview-hq-mp3":"https://freesound.org/data/previews/123/123346_495253-hq.mp3"},"num_downloads":3299,"num_ratings":13},{"id":177110,"name":"hello.wav","duration":10.33,"username":"jleathers292","previews":{"preview-lq-ogg":"https://freesound.org/data/previews/177/177110_3272682-lq.ogg","preview-lq-mp3":"https://freesound.org/data/previews/177/177110_3272682-lq.mp3","preview-hq-ogg":"https://freesound.org/data/previews/177/177110_3272682-hq.ogg","preview-hq-mp3":"https://freesound.org/data/previews/177/177110_3272682-hq.mp3"},"num_downloads":818,"num_ratings":10},{"id":339157,"name":"hello.wav","duration":2.46712,"username":"girlhurl","previews":{"preview-lq-ogg":"https://freesound.org/data/previews/339/339157_5938465-lq.ogg","preview-lq-mp3":"https://freesound.org/data/previews/339/339157_5938465-lq.mp3","preview-hq-ogg":"https://freesound.org/data/previews/339/339157_5938465-hq.ogg","preview-hq-mp3":"https://freesound.org/data/previews/339/339157_5938465-hq.mp3"},"num_downloads":2502,"num_ratings":21}],"previous":"http://freesound.org/apiv2/search/text/?&query=hello&page=1&page_size=10&fields=id,name,previews,duration,num_downloads,username,num_ratings"}
```

Comment:
This is the result of what copying and pasting `https://freesound.org/apiv2/search/text/?fields=id%2Cname%2Cpreviews%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings&page_size=10&page=2&query=hello` into a browser's address bar shows while authenticated.